packages/netlify-cms-core/src/components/MediaLibrary/MediaLibraryModal.js

```diff
@@ -148,15 +149,18 @@ const MediaLibraryModal = ({
               ? t('mediaLibrary.mediaLibraryModal.deleting')
               : t('mediaLibrary.mediaLibraryModal.deleteSelected')
           }
+          downloadButtonLabel="Download"
```

Comment:
The translation API should be used here via the `t` function

Reply:
Should I add different translations of `download` using Google Translate or is the English enough for now ?